data_processing/common/DataStore.py

```diff
@@ -10,14 +10,14 @@
 logger = logging.getLogger(__name__)
 
 class DataStore_v2:
-    def __init__(self):
+    def __init__(self, store_location):
```

Comment:
Just to make sure, store_location includes the /data/ root?

Reply:
store location would be passed in e.g. `/data/some_project/slides/`